src/widgets/visit/visit-button.component.tsx

```diff
@@ -161,16 +161,16 @@ export const StartVisitConfirmation = () => {
   );
 };
 
-const CloseActiveVisitConfirmation = (props: EndVisitProps) => {
+const CloseActiveVisitConfirmation: React.FC<EndVisitProps> = ({
+  currentVisit
+}) => {
   return (
     <div className={styles.visitPromptContainer}>
       <h2>Are you sure to close this visit</h2>
       <p>
-        Visit Type : {props.currentVisit.visitData.visitType.display} Location :{" "}
-        {props.currentVisit.visitData?.location?.display} Start Date :{" "}
-        {dayjs(props.currentVisit.visitData.startDatetime).format(
-          "DD-MMM-YYYY"
-        )}
+        Visit Type : {currentVisit.visitData.visitType.display} Location :{" "}
+        {currentVisit.visitData?.location?.display} Start Date :{" "}
```

Comment:
Wanna fix the text to `Location:{" "}` and `Start Date:{" "}` while we're at it? A space before a colon is not how English is written